medusa/schedulers/utils.py

```diff
@@ -18,9 +18,9 @@
     ('postProcess', 'Post Process', 'post_processor_scheduler'),
     ('subtitlesFinder', 'Subtitles Finder', 'subtitles_finder_scheduler'),
     ('traktChecker', 'Trakt Checker', 'trakt_checker_scheduler'),
-    ('downloadHandler', 'Torrent Checker', 'download_handler_scheduler'),
```

Comment:
@p0psicles 
I think this was removed by mistake?

Reply:
It's there. A few lines up. At least it should be. I'll check